pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/ColumnMetadata.java

```diff
@@ -81,6 +82,7 @@
   private final String dateTimeFormat;
   private final String dateTimeGranularity;
 
+
```

Comment:
Avoid having extra whitespace to classes. Ping me if you need the link for the code format standard, so that whitespace/names are consistently applied by your IDE.